package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "ios-detector",
+  "version": "0.0.0",
+  "author": "Yola Engineering <engineering@yola.com>",
+  "description": "A module that adds a CSS class when it detects the browser is on Apple iOS.",
+  "homepage": "https://github.com/yola/ios-detector",
+  "license": "MIT",
+  "main": "src/ios-detector.js",
+  "typing": "ios-detector.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "karma start"
+  },
+  "keywords": [
+    "Apple",
+    "detection",
+    "detector",
+    "iOS"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yola/ios-detector"
+  },
+  "bugs": {
+    "url": "https://github.com/yola/ios-detector/issues"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "6.3.*",
+    "babelify": "7.2.*",
+    "chai": "3.4.*",
+    "jsgreat": "0.1.*",
```

Comment:
This might be needed as a dependency. Depends on whether anything that references it is packaged.